api_core/setup.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google LLC
+# Copyright 2018, Google LLC
```

Comment:
This differs slightly from the header in go/releasing/preparing (no comma there)